contrib_bots/lib/xkcd/docs.md

```diff
@@ -0,0 +1,37 @@
+# xkcd bot
+
+xkcd bot is a Zulip bot that can fetch comic from xkcd. To use xkcd bot
+you can simply call it with `@xkcd` followed by the command. Like this:
+```
+@xkcd <command>
+```
+xkcd has four commands that is:  
+
+1. `help`  
+This command is used to lists all commands that can be used to use this bot. You
+can use this command by typing `@xkcd help` in stream chat.  
+![](xkcd-help.png)
+
+2. `latest`  
+This command is used to fetch the latest comic from xkcd. You can use this
+command by typing `@xkcd latest` in stream chat.  
+![](xkcd-latest.png)
+
+3. `random`  
+This command is used to fetch random comic from xkcd. You can use this command
+by typing `@xkcd random` in stream chat, then xkcd bot will posts random xkcd's
+comic.  
+![](xkcd-random.png)
+
+4. `<comic_id>`  
+To fetch comic based on the comic id, you can directly use `@xkcd <comic_id>`,
+for example you want to fetch comic with id 1234, then you can fetch it by
+typing `@xkcd 1234`, then xkcd bot will posts xkcd's comic number 1234.  
+![](xkcd-specific-id.png)  
+
+If you type wrong command to xkcd bot, xkcd bot will posts about four commands above.  
+![](xkcd-wrong-command.png)
+
+And if you type wrong comic number, xkcd bot will posts about that xkcd comic with
```

Comment:
`a wrong id`, `will post a message that an xkcd comic strip with that id is not available`